Add a missing linkopts setting to a library, so the subsequent patch
command to set all libraries to static linking will indeed find all
libraries.

This patch is sufficient for Drake's purposes. A patch suitable for
upstreaming would likely import default link flags and use them.

--- absl/time/internal/cctz/BUILD.bazel
+++ absl/time/internal/cctz/BUILD.bazel
@@ -34,6 +34,7 @@ cc_library(
     hdrs = [
         "include/cctz/civil_time.h",
     ],
+    linkopts = [],
     textual_hdrs = ["include/cctz/civil_time_detail.h"],
     visibility = ["//visibility:public"],
     deps = ["//absl/base:config"],
